contracts/profiles/DNSResolver.sol

```diff
@@ -16,6 +17,13 @@ contract DNSResolver is ResolverBase {
     // DNSZoneCleared is emitted whenever a given node's zone information is cleared.
     event DNSZoneCleared(bytes32 indexed node);
 
+    // DNSZonehashChanged is emitted whenever a given node's zone hash is updated.
+    event DNSZonehashChanged(bytes32 indexed node, bytes lastzonehash, bytes zonehash);
+
+    // Zone hashes for the domains.
+    // node => contenthash
+    mapping(bytes32=>bytes) private zonehashes;
```

Comment:
Can you say something about the expected format here? Are they IPFS multicodec values? We should be using the same encoding as we do for the contenthash field, ideally.

Reply:
Good point; added commentary on the format.